website2/docs/compiling-docker.md

```diff
@@ -113,6 +113,39 @@ heron-layer-0.12.0-ubuntu14.04.tar
 heron-tools-0.12.0-ubuntu14.04.tar.gz
 ```
 
+## Set Up A Docker Based Development Environment
+
+In case you want to have a development environment instead of making a full build,
+Heron provides two helper scripts for you. It could be convenient if you don't want
+to set up all the libraries and tools on your machine directly.
+
+The following commands are to create a new docker image with a development environment
+and start the container based on it:
+```bash
+$ cd /path/to/heron/repo
+$ docker/scripts/dev-env-create.sh heron-dev
+```
+
+After the commands, a new docker container is started with all the libraries and tools
+installed. The operation system is Ubuntu 18.04 by default. Now you can build Heron
+like:
+```bash
+\# bazel build --config=ubuntu scripts/packages:binpkgs
+\# bazel build --config=ubuntu scripts/packages:tarpkgs
```

Comment:
Is it worth mentioning the ability to use `--config=ubuntu_nostyle` as a way to cut down on the build time dedicated to static code analysis?

Reply:
Could be helpful to mention in comments I feel. I have never used this config. :D. Please feel free to create a PR for it. Thanks~